src/xdsl/xdsl_opt_main.py

```diff
@@ -66,8 +67,33 @@ def run(self):
         else:
             try:
                 self.apply_passes(module)
+
             except DiagnosticException as e:
-                print(e)
+
+                def gen_frame(e: DiagnosticException, n: int) -> Frame:
+                    return Frame(e, n) if n >= 1 else Frame(e, 3)
+
+                def verbose_frame_output(f: Frame, m: int) -> None:
+                    f.verbose(m) if m > 1 else f.verbose(1)
+
+                if self.args.verbose_trace is not None:
+                    if self.args.verbose_trace == []:
```

Comment:
Is it actually possible to trigger this line? Doesn't the mean that no arguments were given to the `verbose_trace` flag? If so, giving no arguments fails with `xdsl-opt: error: argument --verbose-trace: expected 2 arguments`.